### DISCUSSION TITLE
Load Reactions in MoBI following esqlabs tutorials

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1363] [Models]
Hi everyone

When I try to load reaction in MoBi following the tutorial steps but something goes wrong.  The notification shown the warning information as "Reaction ' ADC_TMEFF2 _internalization' was not created. One reactant is either not present and/or a matching physical container was not found". I  can’t find out where  the mistake happened.

Any help would be very appreciated!

**Comment by maalbrec89** [comment]
Hi,

If you load the new reaction network, it wants to attach to the reactions you already have in there. Could you check whether you have no typos in naming the reactions? Maybe you forgot a dot?

For example:
On the slide: 'Compound: ADC'
you have the entry:
4. Data source is 'Boswell et al.' 
(Check your spelling and the dot! You will upload reactions requiring identical labels)

Congratulations. You seem to be the first to apply the newest tumor lecture version.

**Reply by Supercn01** [reply]
Hi
I really appreciate your help！
I didn't notice the importance of the dots. This issue was resolved with your kind help. 
Thanks again

**Reply by Supercn01** [reply]
Hi 

I met new problem. When I create simulation follow the tutorial step to the section of  "SIMULATE THE ADDED TUMOR" but some errors  have occurred.  The warning information shown in the image. What details in the tutorial did I miss from paying attention to these errors?

I will appreciate your kind help!
<img width="829" alt="屏幕截图 2022-10-28 144214" src="https://user-images.githubusercontent.com/85626813/198573859-836d98a7-4aa2-421f-bcb5-5cc0894b2df0.png">

**Reply by maalbrec89** [reply]
Unfortunately, it can happen that one makes a small mistake and this may have happened where you set reactions or passive transporters. You can:

1. Just proceed with the check-up point files, which you can download on the top right
2. You can compare your file with the check-up point file and look whether you find differences
3. You redo it from the last simulation that worked and take your time to work carefully on this.

I know, it is annoying but also I and the best of us may just have to redo a course section from time to time as this is faster than tracking down the error.

**Reply by Supercn01** [reply]
Thanks  for your kind help. I will redo the course section more carefully to avoid mistakes.
